src/moh_data/main.py

```diff
@@ -31,6 +34,110 @@ def _run(self):
         self._total_arrival = self._excel_file.get_daily_arrival_sum()
         self._total_overseas_reported_date = self._excel_file.get_overseas_reported_sum()
 
+    def get_confirmed_cases_on(self, date):
+        """
+        Get the number of confirmed cases on a particular date
+
+        :param date: list with (month, day) format e.g. (04, 14).
+        :return:
+        """
+        return self._get_cases_on(self._total_daily_confirmed, date)
+
+    def get_confirmed_cases_between(self, dates):
+        """
+        Get the number of confirmed cases between two dates
+
+        :param dates: list of lists with ((month, day), (month, day)) format e.g. ((04, 08), (04, 14)).
+        :return:
+        """
+        return self._get_cases_between(self._total_daily_confirmed, dates)
```

Comment:
I guess it's fine to have those methods, but the SEIR model only knows about days not dates. So, it would be nice to also have something like `get_confirmed_cases_on(self, day)` with `day` being an integer. Similarly, and for consistency, we might also want to have something like `get_confirmed_cases_between(from_day, to_day)` although I would have no use for it personally.

The same goes for the other `get_xxx()` methods.

Reply:
FWIW, the SEIR model might only need the `get_cumulative_xxx()` methods, but the other `get_xxx()` methods may be useful to others, I don't know.

Reply:
I see. So we would assume that `day == 1` would be the first date that we have the data available for? I'll add these to the script.

Reply:
Actually, maybe this should be considered as day 0...? This would thus become our initial conditions?

Reply:
That makes sense.